pyroclast/common/plot.py

```diff
@@ -63,6 +63,8 @@ def plot_images(image_tensors, row_labels=None, col_labels=None):
     nrows = len(image_tensors)
     ncols = max([len(x) for x in image_tensors])
 
+    kwargs = {'cmap': cmap}
```

Comment:
Does this get added to? Why not pass the dictionary directly in without naming it?

Reply:
It might. That's why I did some filtering here. Can think about it again later.